config/examples/database.yml

```diff
@@ -43,6 +47,9 @@ base: &default
   encoding: utf8
   pool: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   collation: utf8_bin
+  sslca: <%= ENV['DATABASE_SSL_CA'] %>
+  sslcert: <%= ENV['DATABASE_SSL_CERT'] %>
+  sslkey: <%= ENV['DATABASE_SSL_KEY'] %>
```

Comment:
Same question here? Have you tried setting these in the the connection URI? Probably you did but have to ask :)

Reply:
Yeah, sure... of course I tried... https://github.com/3scale/zync/pull/532#discussion_r1682357224